### PR TITLE
feat(platform/m1-macstadium): Run tests on M1 Mac

### PR DESCRIPTION
**Pull request by janpio**
via SSH to MacStadium M1

(failing tests were also on `dev`, so can be ignored.)

### CODE REVIEW

**Review by janpio — COMMENTED** [review]
Make sure to review the actual test output please.

File: platforms/m1-macstadium/code/index.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const { PrismaClient, Prisma } = require('@prisma/client')
+
+const prisma = new PrismaClient()
+
+describe('Prisma', () => {
+  afterAll(() => {
+    prisma.$disconnect()
```
Comment (Jolg42) [inline platforms/m1-macstadium/code/index.test.js:7]: Could use an await probably

**Review by Jolg42 — COMMENTED** [review]
Looks ok

File: platforms/m1-macstadium/code/index.test.js (hunk 1 above)
Comment (janpio) [reply platforms/m1-macstadium/code/index.test.js:7]: True, will fix via unrelated PR.

**Pull request merged**